app/javascript/kramdown.jsx

```diff
@@ -65,14 +64,24 @@ export default function createKramdownExtensionEditorComponent(options) {
   );
 
   // Convert a regex match back to an object whose keys are the field names and values are the field values
-  const fromBlock = (match) =>
-    fields.reduce(
-      (result, field, index) => ({
-        ...result,
-        [field.name]: unescapeExtensionAttribute(match[index + 1]),
-      }),
-      {}
-    );
+  const fromBlock = (match) => {
+    const attrs = match[1];
+    const rx = new RegExp(attrRegex, "gi");
+
+    const parsed = {};
+
+    for (let m = rx.exec(attrs); m; m = rx.exec(attrs)) {
+      const attrName = m[1];
+      const attrValue = m[2];
+      parsed[attrName] = unescapeExtensionAttribute(attrValue);
+    }
+
+    return fields.reduce((result, field) => {
+      // eslint-disable-next-line no-param-reassign
+      result[field.name] = parsed[field.name] ?? "";
```

Comment:
[question] any reason you didn't continue with the `{...result, [field.name]: (parsed[field.name] ?? "")}` pattern from the last version?

Reply:
Good catch, yeah, I should do that!